recipes/ggd/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+rm -rf tests
+
+$PYTHON setup.py install
```

Comment:
(NB: I'm no fan of the "Python special handling" via `${PYTHON} ...` `"{{ PYTHON }} ..."`, but it's common convention, I think. At least on conda-forge it's recommended, can't remember if we do that on Bioconda, too..)

Reply:
@mbargull, thanks for the suggestions, however, this convention is pretty standard and as such I am going to keep it this way.

Reply:
@mikecormier we do have reasons for those recommendations :)
Please stick to them if you want your recipe be part of Bioconda.

Reply:
@bgruening, I understand the reasons for recommendations, however, the recommended is unwarranted as the current install is a convention held in conda and bioconda, and is the traditional way to use a setup command.

Reply:
> as the current install is a convention held in conda and bioconda

No:
```sh
grep -rh 'setup.py install\|pip install' bioconda-recipes/recipes | sed 's/^\s*-\?\s*//' | sort | uniq -c | sort -n | tail
     20 script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
     20 script: "{{ PYTHON }} -m pip install . --no-deps -vv"
     21 $PYTHON -m pip install . --ignore-installed --no-deps -vv
     21 script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
     23 $PYTHON setup.py install
     28 script: python setup.py install --single-version-externally-managed --record=record.txt
     83 script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
    111 script: python -m pip install --no-deps --ignore-installed .
    181 $PYTHON setup.py install --single-version-externally-managed --record=record.txt
    225 script: {{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv
```
We can see that there are recipes with only `setup.py install`, but only few and they should probably be changed.
The most prevalent conventions are `pip install ...` or `setup.py install --single-version-externally-managed --record=record.txt` and as @bgruening mentioned, there are reasons for this.
Some reasons:
1. Additional parameters to `setup.py` are to prevent `setuptools` to download additional dependencies (which would erroneously be added to the package we are trying to build).
   (In practice you'll likely get a build failure because `conda-build` sets things up to error out when such downloads would happen -- but this should only be regarded as a safety net, IMO.)
2. `pip install` you'll get nicer entry point scripts which do not enforce a direct run dependency on `setuptools`.
   (`setuptools` entry points use `pkg_resources` which is part of `setuptools` itself.)
   (`conda-build` sets up some environment variables for `pip`, too. If you think that might be a little non-transparent (which I might agree with), you can use `pip install . --no-deps --ignore-installed -vv`, too)

Reply:
Thanks for the suggestions and info @mbargull. I have updated the recipe to use `script: python -m pip install . --no-deps --ignore-installed -vv` in the meta.yaml file and have removed the build.sh